package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@coseeing/access8math-web-lib",
-  "version": "1.0.5",
+  "version": "2.0.0",
```

Comment:
I upgraded the version to 2.0.0 because the LINK block feature has been removed. Let me know if this is not appropriate!